### ISSUE TITLE
Part 49 - Using PHP to Handle a Clean URL

### ISSUE THREAD

**Issue by angelanjali** [thedigicraft/Atom.CMS#96]
I am loosing css of the page at this point: blog/new-jquery-plugin
If there is only one call part, then it is okay.
Can anybody tell what can be the reason behind. I have copied exactly the same code and steps as shown in the vedio.

![atomcms](https://cloud.githubusercontent.com/assets/10372925/6020500/a5320c9e-abd2-11e4-8229-4c4c7190ff62.png)

**Comment by creptor** [comment]
this site was made to be in just one folder (or one folder away from the index.php). So if you don't have your index.php file localhost/dynamic/blog/ you can recreate it there and it will work. If that's not your problem, please post your index.php and setup.php(or whathever you called your starter web config)

**Comment by creptor** [comment]
NOTE: the CDN for bootstrap and other things must have an http: or https: defined for localhost

**Comment by thedigicraft** [comment]
yes as @creptor says; make sure you have http:// rather than the // that comes with the CDN when you copy/paste it.
